doozer/doozerlib/cli/images_streams.py

```
@@ -81,6 +81,10 @@ def get_image_digest(pullspec: str, registry_config: Optional[str] = None) -> Op
 
     rc, stdout, stderr = exectools.cmd_gather(cmd)
     if rc != 0:
+        # #region agent log
+        _logger = logging.getLogger(__name__)
```

Ruff (0.15.18)

[error] Undefined name `logging`

(F821)